flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/batch/sql/agg/AggregateReduceGroupingTest.xml

```diff
@@ -765,13 +765,13 @@ HashAggregate(isMerge=[false], groupBy=[a1], auxGrouping=[d1], select=[a1, d1, C
   </TestCase>
   <TestCase name="testSingleAggOnlyConstantGroupKey">
     <Resource name="sql">
-      <![CDATA[SELECT count(c1) FROM T1 GROUP BY 1, true]]>
```

Comment:
At first I was wondering: Why changing the integer to string?
Does not seem to be necessary in this PR, as this test is testing grouping by a constant.

But now I understand that those `1`s now take a different meaning and I like the approach to simply change it to a string 👍